source/Calamari.Testing/Helpers/CalamariResult.cs

```diff
@@ -59,70 +56,28 @@ public void AssertOutputVariable(string name, IResolveConstraint resolveConstrai
             Assert.That(variable, resolveConstraint);
         }
 
-        public void AssertServiceMessage(string name, IResolveConstraint? resolveConstraint = null, Dictionary<string, object>? properties = null, string message = "", params object[] args)
```

Comment:
I split this method into 3, as having `resolveConstraint` null was... very odd. `Assert.That` expected it to be not null, so I'm not sure how it was working 🤷 

Splitting it up made it obvious that parts were unused, so they got the ✂️